test/test/recovery/RecoveryTests.java

```diff
@@ -39,6 +39,11 @@ public void stringBuilderArm(TestProcess p) throws Exception {
     @Test(mainClass = Numbers.class, debugNonSafepoints = true)
     public void numbers(TestProcess p) throws Exception {
         Output out = p.profile("-d 3 -e cpu -o collapsed");
+        if (System.getProperty("os.arch").equals("ppc64le")) {
+            // the test behaves differently on PowerPC
```

Comment:
I wonder why it works differently? Isn't it a real profiler issue?
If not, then the test probably makes a wrong assumption and needs to be adjusted for other archs, too.

Reply:
The stubs don't appear on PPC. So yes, that might be a profiler issue.